packages/react/src/components/dropdown-list/dropdown-list.test.tsx

```diff
@@ -494,6 +494,26 @@ describe('Dropdown list', () => {
         expect(container.firstChild).toMatchSnapshot();
     });
 
+    describe('onClose callback', () => {
```

Comment:
Pas besoin de ce describe là, je mettrais le test dans la section `opening and closing the listbox`